src/common/snippets/src/op/subgraph.cpp

```diff
@@ -351,6 +352,8 @@ ov::PartialShape snippets::op::Subgraph::canonicalize(const BlockedShapeVector&
     // to align precision inside Subgraph body that is supported by Plugin
     align_element_types(outputShapes, inputShapes);
 
+    post_action.run_passes(body_ptr());
```

Comment:
We cannot call here `target_generator->get_supported_precisions()` because we haven't converted the ops to snippet dialect. The part of nodes in current state of pipeline doesn't have the corresponding emitter yet. For example, we have just `MatMul` here and `CPUTargetGenerator` doesn't know anything about this node.

Need to discuss offline

Reply:
discussed, thanks